### PR TITLE
Added deploy steps to Azure Pipeline YAML

### PR DESCRIPTION
**Pull request by vaquierm**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -47,6 +51,21 @@ jobs:
...
+        displayName: 'Deploy Common Package'
+        condition: and(succeeded(), eq(variables.isPR, false))
+        env:
+          HOME: '$(Agent.BuildDirectory)'
+          BRANCH: '$(Build.SourceBranchName)'
+          PULL_REQUEST: '$(isPR)'
+          COMMIT_MESSAGE: '$(Build.SourceVersionMessage)'
+          PACKAGE_LOCATION: 'packages/common'
+          PACKAGE_NAME: 'common'
+          SITE_NAME: 'script-lab-react-storybook'
```
Comment (raulclouse) [inline azure-pipelines.yml:64]: do we need this?  We aren't using Storybook any longer...

File: azure-pipelines.yml [hunk 2]
```diff
@@ -47,6 +51,21 @@ jobs:
...
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_ALPHA: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_ALPHA)'
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_BETA: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_BETA)'
+          DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_STAGING: '$(DEPLOYMENT_PASSWORD_SCRIPT_LAB_REACT_STORYBOOK_STAGING)'
+
   - job: EditorPipeline
```
Comment (raulclouse) [inline azure-pipelines.yml:69]: Seems like a lot of duplicate code in these jobs.  Couldn't we use Templates and parameters to factor this out?  https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#template-references

File: azure-pipelines.yml (hunk 1 above)
Comment (vaquierm) [reply azure-pipelines.yml:64]: What is the site name that this is deployed to? I still see storybook in our portal

File: azure-pipelines.yml (hunk 2 above)
Comment (vaquierm) [reply azure-pipelines.yml:29]: I didn't know that was possible. Thanks for the resource Raul, that's super helpful to make the pipeline a lot cleaner

File: azure-pipelines.yml (hunk 1 above)
Comment (lexahall) [reply azure-pipelines.yml]: I think this is a bit misleading and that we may actually still deploy common to this site, but that the name doesn't make a lot of sense anymore. I dont think this is actually tied to storybook

**Review by lexahall — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (vaquierm) [reply azure-pipelines.yml]: I think Lexa is right. From my understanding the site name we use just happens to have the word storybook. Closing for now, and if we feel like it we can rename all that later

**Pull request merged**